targets/TARGET_STM/TARGET_STM32L4/device/stm32l4xx_hal.c

```diff
@@ -404,6 +404,18 @@ uint32_t HAL_GetDEVID(void)
 }
 
 /**
+  * @brief Return the unique device identifier (UID based on 96 bits)
+  * @param UID: pointer to 3 words array.
+  * @retval Device identifier
+  */
+void HAL_GetUID_12Byte(uint32_t *UID)
+{
```

Comment:
Maybe should you rather name the function with an 's' ?  HAL_GetUID_12Bytes ?

Reply:
@LMESTM thanks for comments. I'll rename and move it to other place.